servers/src/main/java/tachyon/master/lineage/journal/LineageEntry.java

```diff
@@ -42,6 +43,33 @@
   private final String mJobOutputPath;
   private final long mCreationTimeMs;
 
+  /**
```

Comment:
Either provide parameter descriptions or remove the function-level JavaDoc (so that mvn site does not complain).